index.html

```diff
@@ -2,86 +2,49 @@
 	<head>
 		<title>RCDC - Home</title>
 		
-		<meta charset='utf-8' />
-		<link href="Style/index.css" rel='stylesheet' />
-		<link href="Style/banner.css" rel='stylesheet' />
+		<meta charset="utf-8"/>
+        <meta http-equiv="content-type" content="text/html">
+        <meta name="viewport" content="width=device-width,user-scalable=no,initial-scale=1">
```

Comment:
You have to add some boilerplate like this to make it work on mobile. Weird, huh?